docs/src/transforms.md

```diff
@@ -506,6 +506,12 @@ This is mainly used in conjunction with the `TuneableConsistencyScatter` transfo
 This transform sends messages to both the defined sub chain and the remaining down-chain transforms.
 The response from the down-chain transform is returned back up-chain but various behaviours can be defined by the `behaviour` field to handle the case when the responses from the sub chain and down-chain do not match.
 
+Tee also exposes an optional HTTP API to switch which chain to use as the "result source", that is the chain to return responses from.
+
+`GET` `/transform/tee/result-source` will return `"regular-chain"` or `"tee-chain"` indicating what chain is being used for the result source.
```

Comment:
`"regular-chain"` -> `regular-chain`, otherwise people might try to send the string "\\"regular-chain\\""